### PR TITLE
Wider SQL updates re-alignment

### PR DESCRIPTION
**Pull request by graham73may**
**Summary of changes:** 
1. Re-aligning Query 1 autocomplete SQL to match how the PHP is dealing with multiple words. 
2. Re-aligning Query 2 autocomplete SQL to match how the PHP is dealing with multiple words. 
3. ~Adding Query 2 autocomplete SQL for when "Include related roles" is checked.~
   - ~Need to check if this is style required?~
   - ~See: https://app.productive.io/27782-soapbox-communications/projects/436389/tasks/task/7819603?filter=NjkzMjE1&taskActivityId=129450486~
4. Query 2 role lookup - Use same placeholder for both parts of the union
5. Added intellij to gitignore (hope you don't mind!)
6. `normalized_name` added to Query 1 autocomplete query

### CODE REVIEW

File: sql/2_persons_holding_appointment/2_role_id_related.sql
Comment (philipnye) [inline sql/2_persons_holding_appointment/2_role_id_related.sql:1]: You've correctly picked up on inconsistency I've introduced in this file - one use of `@role` and one of `@display_name`. It makes sense to standarise, but I wonder if it makes more sense to standardise to `@display_name`, which is as used in https://github.com/instituteforgov/ifg-ministers-database-public/blob/07ac82a37c0207a10aa6052d09b8684d7b19528a/sql/2_persons_holding_appointment/2_role_id_simple.sql#L5?

**Review by philipnye — COMMENTED** [review]
One small suggestion - see what you think

File: sql/2_persons_holding_appointment/2_role_id_related.sql
Comment (graham73may) [reply sql/2_persons_holding_appointment/2_role_id_related.sql:1]: Happy to use whichever you prefer!

Comment (graham73may) [reply sql/2_persons_holding_appointment/2_role_id_related.sql:1]: Pushed an update to use `@display_name`

**Pull request merged**